src/components/mapgl/model/mapLayers.ts

```diff
@@ -47,8 +47,10 @@ export interface Style {
   id: string;
   name: string;
   layerIds: Set<string>;
+  geometries?: Array<string>;
   geomStrategy?: geomStrategyEnum;
   isDefault?: boolean;
+  isHidden?: boolean;
```

Comment:
`isHidden` is not clear here, maybe `isAvailable` or `active` would be better

Reply:
Fixed